.circleci/config.yml

```diff
@@ -648,7 +648,7 @@ jobs:
           command: docker run -t --gpus all -v $PWD:$PWD -w $PWD -e UPLOAD_CHANNEL -e CU_VERSION "${image_name}" .circleci/unittest/linux/scripts/install.sh
       - run:
           name: Run tests
-          command: docker run -t --gpus all -v $PWD:$PWD -w $PWD "${image_name}" .circleci/unittest/linux/scripts/run_test.sh
+          command: docker run -e CIRCLECI -t --gpus all -v $PWD:$PWD -w $PWD "${image_name}" .circleci/unittest/linux/scripts/run_test.sh
```

Comment:
Since we're using `docker run`, we have to add that so that the docker container knows about the `CIRCLECI` env variable (set by CircleCI).

We do that in a bunch of places already.

A more robust version will be to maybe use the `BASH_ENV` variable, or to dump `env` into a file and source it in the container. We'll have to do that soon enough when we expand the `cpu_and_gpu()` to also work with CircleCI, but for now this is good enough as only the linux_gpu job needs it